src/platform/packages/shared/kbn-grouping/src/containers/query/index.ts

```diff
@@ -25,17 +25,17 @@ export const MAX_RUNTIME_FIELD_SIZE = 100;
  * Composes grouping query and aggregations
  * @param additionalFilters Global filtering applicable to the grouping component.
  * Array of {@link BoolAgg} to be added to the query
- * @param from starting timestamp
- * @param groupByFields array of field names to group by
```

Comment:
Seems like this param docs got lost in the change

nit: these JSDocs could be moved to the [GroupingQueryArgs](https://github.com/elastic/kibana/blob/1b9bf801e485e5193a51c391c5b8c7769713c7a0/src/platform/packages/shared/kbn-grouping/src/containers/query/types.ts#L34) type, just above every parameter type definition

Reply:
Hey, these ones have been removed because they are not used anymore 😊